content/100-getting-started/02-setup-prisma/200-add-to-existing-project/110-relational-databases/170-baseline-your-database.mdx

```diff
@@ -212,6 +212,13 @@ The command will mark `0_init` as applied by adding it to the `_prisma_migration
 
 You now have a baseline for your current database schema. To make further changes to your database schema, you can update your Prisma schema and use `prisma migrate dev` to apply the changes to your database.
 
+[!NOTE]
```

Comment:
@ruheni Maybe you can suggest the needed "admonition" code here.

Reply:
https://github.com/prisma/docs/pull/5387/files#r1372212071